src/devices/tuya.ts

```diff
@@ -5503,6 +5503,30 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        model: 'TS0603',
```

Comment:
- Could you provide a link to this device?
- Do you have a better model number? Is there something written on the device?

Reply:
bought it on aliexpress: https://aliexpress.com/item/1005006776475188.html
the only text on the Device is QS-Zigbee-C03 but all i found on this name is a curtain motor controller.

Reply:
Then let's leave it as is, thanks!